### PR TITLE
changed sh scripts for community release pipeline

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
please merge together with. https://github.com/kiegroup/kie-jenkins-scripts/pull/590

File: script/release/08a_communityPushTags.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
 # create a tag
-commitMsg="Tagging $tagName"
-./droolsjbpm-build-bootstrap/script/git-all.sh tag -a $tagName -m "$commitMsg"
+commitMsg="Tagging $kieVersion"
+./droolsjbpm-build-bootstrap/script/git-all.sh tag -a $kieVersion -m "$commitMsg"
 
 # pushes tag to github ¡/kiegroup
-./droolsjbpm-build-bootstrap/script/git-all.sh push origin $tagName
\ No newline at end of file
+./droolsjbpm-build-bootstrap/script/git-all.sh push -n origin $kieVersion
```
Comment (mareknovotny) [inline script/release/08a_communityPushTags.sh]: Is `-n` intentionally added? that means `dry-run` execution

Comment (mbiarnes) [reply script/release/08a_communityPushTags.sh]: nop - I have to remove this - good spot (this was done for testing)

Comment (mbiarnes) [reply script/release/08a_communityPushTags.sh]: fixed!

**Pull request closed without merging**